assembl/views/api2/discussion.py

```diff
@@ -540,6 +540,13 @@ def extract_taxonomy_csv(request):
             message = "no message"
         if not message:
             message = "no message"
+
+        if thematic == "no thematic associated":
+            idea_ids = m.Idea.get_idea_ids_showing_post(query.id)
+            for thematic_id in reversed(idea_ids):
+                if db.query(m.Idea).filter(m.Idea.id == thematic_id).first().title:
```

Comment:
you could put `db.query(m.Idea).filter(m.Idea.id == thematic_id).first().title` in a variable here for readability because you reuse it